citi-onboarding/Eventos-CITi
# Feat/create blog model

## What I did

- Create Blogpost.js file inside models folder at server
- Insert the blog model at blogpost.js
- Insert the source /api/blogpost at index.js
- Fixed the file Post.js

## How to test

1. Open the terminal and go to the folder of this project

1. In terminal:
   1. `cd server/`
   1. `node index.js`
  


1. In your browser
   1. Open localhost [http://localhost:3001/admin/blog-posts](url)
   1.  Then press Blog Post
   1. Check if image, date, description and link fields are available
   1. Then, save the model
   1. Go back to [http://localhost:3001/admin/blog-posts](url)
   1. Check if the blog archive has the correct title (id)
   1. Go to [http://localhost:3001/api/blogpost](url)
   1. Check if error 500 is not showing up
   1. Check if the code is showing up

&nbsp;
*
